src/cli/utils/async-load-config.ts

```diff
@@ -0,0 +1,22 @@
+import { ConfigModule } from 'core/types';
+import { getConfigFile } from 'medusa-core-utils';
+
+export async function asyncLoadConfig(): Promise<ConfigModule> {
```

Comment:
would be nice to pass the path as the argument of the function `configPath: string`

Reply:
we could do that, but by default medusa expects the configfile path to be in the root folder, I didn't want to change that behaviour

Reply:
It would just give more flexibility and would make it explicit, the same way you have to pass the path to medusa config loader. Wdyt?